src/libYARP_OS/include/yarp/os/Timer.h

```diff
@@ -54,22 +54,22 @@ struct YARP_OS_API yarp::os::YarpTimerEvent
 
 struct YARP_OS_API yarp::os::TimerSettings
 {
-    TimerSettings(double inRate) :
-            rate(inRate),
+    TimerSettings(double inPeriod) :
```

Comment:
This parameter was expressed in milliseconds and not it is expressed in seconds?

Reply:
I see, it is actually related to https://github.com/robotology/yarp/pull/1718/files#diff-598e25a1a938b9208e8bb83d7e136a7fL192 . Ok for me!